src/qibocal/web/report.py

```diff
@@ -26,3 +27,24 @@ def create_report(path, actions=None):
 
     with open(os.path.join(path, "index.html"), "w") as file:
         file.write(html)
+
+
+def create_autocalibration_report(path, history):
+    """Creates an HTML report for the data in the given path."""
+
+    filepath = pathlib.Path(__file__)
```

Comment:
Make this a top-level constant.
```suggestion
    WEB_DIR = pathlib.Path(__file__).parent
```